test2.js

```diff
@@ -0,0 +1,19 @@
+var foo = '' ;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26923)
```suggestion
var foo = "" ;
```